shuttleofx_catalog/views.py

```diff
@@ -378,7 +385,9 @@ def getPlugin(pluginRawIdentifier, pluginVersion="latest", bundleId=None):
 
     plugin = plugins[0]["plugin"]
 
-    return mongodoc_jsonify(plugin)
+    logging.error(plugin);
```

Comment:
Is this really useful ? It looks like ```plugin``` is always logged no matter if there's an error or not, thus the logs are kind of messy :confused: